documentation/configure.markdown

```diff
@@ -180,6 +180,8 @@ When this is set to true, debugging information will be written to `<vardir>/deb
 
 ### `url-prefix`
 
+> **Deprecated:** This setting will be removed in the future and url-prefixes should be configured using the `web-router-service`.
```

Comment:
In other cases we duplicate the docs from there. Unfortunately TK docs aren't currently being published to the docs site (is that still the case @cprice404 ?). I think web routing stuff is painful to write in ini, is that right? The rest of the docs are in ini, probably makes sense to keep it that way unless we switch all of it over.